### ISSUE TITLE
Move fabtests out of an "AppStream" workflow

### ISSUE THREAD

**Issue by hroncok** [minimization/content-resolver-input#1191]
Considering https://github.com/minimization/content-resolver-input/pull/1164#issuecomment-2115771123 and https://issues.redhat.com/browse/ENGCMP-4168 I'd like fabtests not to be listed in a workflow that is called "RDMA Stack **Appstream** Packages"

**Comment by yselkowitz** [comment]
Then please file a PR to create a new workflow with fabtests in it, and remove fabtests from the "appstream" workflow.

**Comment by hroncok** [comment]
Yes, the maintainers of this workflow should do that. I asked them to do that 5 months ago in https://issues.redhat.com/browse/RHEL-12293

Instead, the package was removed entirely and then the change got reverted in https://github.com/minimization/content-resolver-input/pull/1164 saying it is a temporary solution. Since nothing changed in another month, I opened this ticket.
